packages/api/src/parse-cli.ts

```diff
@@ -453,6 +456,73 @@ export default function parseCli(argv?: string | readonly string[]) {
         type: "string",
         default: "127.0.0.1:7935",
       },
+      "default-stream-profiles": {
+        describe: "default stream transcoding profiles if none are provided",
+        type: "string",
+        default: JSON.stringify([
+          {
+            name: "240p0",
+            fps: 0,
+            bitrate: 250000,
+            width: 426,
+            height: 240,
+            profile: "H264ConstrainedHigh",
+          },
+          {
+            name: "360p0",
+            fps: 0,
+            bitrate: 800000,
+            width: 640,
+            height: 360,
+            profile: "H264ConstrainedHigh",
+          },
+          {
+            name: "480p0",
+            fps: 0,
+            bitrate: 1600000,
+            width: 854,
+            height: 480,
+            profile: "H264ConstrainedHigh",
+          },
+          {
+            name: "720p0",
+            fps: 0,
+            bitrate: 3000000,
+            width: 1280,
+            height: 720,
+            profile: "H264ConstrainedHigh",
+          },
+        ] as FfmpegProfile[]),
+        coerce: (str: string): FfmpegProfile[] => {
```

Comment:
**nit:** Mind moving this to the top of the file with a descriptive name, like the other `coerce...` funcs around there? Or perhaps even just adjust `coerceJsonArray` to receive the validator as an optional argument 👀

Reply:
Done!